doc/tutorials/using_vxworks_as_uos.rst

```diff
@@ -0,0 +1,137 @@
+.. _using_vxworks_as_uos:
+
+Using VxWorks as User OS
+########################
+
+This tutorial describes how to run VxWorks as the User OS on the ACRN hypervisor. We are using Kaby Lake-based
+NUC (model NUC7i5DNHE) in this tutorial. Other :ref:`ACRN supported platforms <hardware>` should work as well.
+
+Introduction to VxWorks
+***********************
+
+VxWorks is a real-time operating system (RTOS) that can be used in embedded systems. It can be customized easily
+and run on most processors designed for distributed computing.
+
+Steps for Using VxWorks as User OS
+**********************************
+
+#. Build VxWorks
+
+   Follow the `VxWorks Getting Started Guide <https://docs.windriver.com/bundle/vxworks_7_tutorial_kernel_application_workbench_sr0610/page/rbu1422461642318.html>`_
+   to setup the VxWorks development environment and build the VxWorks Image.
+
+   .. note::
+      The following kernel configuration should be excluded:
+        * INCLUDE_PC_CONSOLE
+        * DRV_SIO_PCI_NS16550
+        * SELECT_PC_CONSOLE_KBD
+
+      The following kernel configuration should be included:
+        * DRV_SIO_IA_NS16550
+        * DRV_SIO_FDT_NS16550
+
+      The following kernel configuration value should be set:
+        * CONSOLE_BAUD_RATE = 115200
+        * SYS_CLK_RATE_MAX = 1000
+
+#. Build GRUB2 BootLoader Image
+
+   We use grub-2.02 as the bootloader of VxWorks in this tutorial. You may gave other version of GRUB2 a try.
+   We can download the grub-2.02 from `ftp://ftp.gnu.org/gnu/grub/grub-2.02.tar.gz`.
+
+   .. code-block:: none
+
+      $ zcat grub-2.02.tar.gz | tar xvf -
```

Comment:
How about this single-line to download and extract Grub 2.02 sources?
```
curl ftp://ftp.gnu.org/gnu/grub/grub-2.02.tar.xz | tar xJv
```

Reply:
Great.